Updates to Usage report (#2010)
RedisLabs/redislabs-docs: content/rc/administration/usage-reports.md

categories: ["RC"]
aliases: /rv/administration/usage-reports/
---

> The **Usage Report** can be used to view the daily memory usage and shard usage of all databases from the subscriptions associated with your account.

Reviewer: ```suggestion
The **Usage Report** shows the daily memory usage and shard usage of all databases from the subscriptions associated with your account.
```

![usage_report](/images/rc/usage-report-memory-usage.png)

> You can filter the data by using:
> * The **View Statement For** menu and selecting a month/year to view the daily memory usage during a specific month
> * The **Subscription** menu and selecting a subscription to view the daily memory usage of a specific subscription associated with you account
> * The **Database** menu and selecting a database to view the daily memory usage of a specific database

Reviewer: ```suggestion
To filter the data, you can:
* Select a month and year from the **View Statement For** list to view the daily memory usage during a specific month.
* Select a subscription from the **Subscription** list to view the daily memory usage of a specific subscription associated with your account.
* Select a database from the **Database** list to view the daily memory usage of a specific database.
```
